app/controllers/service_controller.rb

```diff
@@ -141,6 +141,15 @@ def service_form_fields
 
   private
 
+  def sanitize_output(stdout)
```

Comment:
The substitution needs to change: `u\"{{ manageiq_mach | default('localhost') }}\"` into `\{\{ manageiq_mach | default(\&#39;localhost\&#39;) \}\}`